docker/postgres/setup_db.sql

```diff
@@ -0,0 +1,45 @@
+CREATE USER "power_user" WITH
```

Comment:
Let's add a short comment at the top of this script describing what it's doing and how it might need to be customized for use in production.

Reply:
Sure, but I would defer that to you.

Take note this was copied from `scripts/test_db.sql` in the zen repo to get the setup to work.